docs/features/generate-image-action.md

```
@@ -0,0 +1,51 @@
+ ## Summary
```

markdownlint-cli2 (0.22.1)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)